Docs: Fix formatting and clarity for SSO Setup (#288)
CluedIn-io/CluedIn.Documentation: docs/020-deployment/howtos/020-setup-sso.md

Redirect URI is the location to which the Microsoft identity platform redirects the user's client and sends security tokens after authentication.

> In the setup below, we will be working with fictitious domain 'yourdomain.com' and the two subdomains 'app' and 'departmentX'. 
> 
> **departmentX** references the main CluedIn UI you would access.  
> And **app** references the backend application. By default, it is just app, but can be changed in the values file.
> 

Reviewer: ```suggestion
In the setup below, we will be working with fictitious domain 'yourdomain.com' and the two subdomains:

- 'app' – references the main CluedIn UI you would access.

 - 'departmentX' – references the back-end application. By default, it is just app, but can be changed in the values file.

```

Author: I have amended this as there was a leading space, and i think bullet points are find to be together

Author: @SolomiiaSnihur are you happy with amendment above?

**To add redirect URIs**

1. In the Azure portal, in **App registrations**, select your application.
1. Select **Authentication** > **Add a platform**.
1. In the right pane, select **Web**.

    ![Add_redirect_URIs_Configure_platforms.png](../../assets/images/ama/howtos/configure-sso-5.png)

1. In the **Configure Web** pane, specify the following:

>     1. In **Redirect URIs**, add a redirect URI for your application.
> 
>         ![Add_redirect_URIs_Redirect_URIs.png](../../assets/images/ama/howtos/configure-sso-6.png)  
>         `https://app.yourdomain.com`
> 
>     1. In **Front channel logout URI**, add a logout URL for your application.
> 
>         ![Add_redirect_URIs_Logout_URL.png](../../assets/images/ama/howtos/configure-sso-7.png)  
>         `https://app.yourdomain.com/logout`
> 
>     1. In the **Implicit grant and hybrid flows** section, select the **ID tokens** checkbox, and then select **Configure**.
> 
>         ![Add_redirect_URIs_ID_tokens.png](../../assets/images/ama/howtos/configure-sso-8.png)
> 

Reviewer: ```suggestion
    1. In **Redirect URIs**, add a redirect URI for your application.

        `https://app.yourdomain.com`
        
        ![Add_redirect_URIs_Redirect_URIs.png](../../assets/images/ama/howtos/configure-sso-6.png)

    1. In **Front channel logout URI**, add a logout URL for your application.

        `https://app.yourdomain.com/logout`
        
        ![Add_redirect_URIs_Logout_URL.png](../../assets/images/ama/howtos/configure-sso-7.png)  

    1. In the **Implicit grant and hybrid flows** section, select the **ID tokens** checkbox, and then select **Configure**.

        ![Add_redirect_URIs_ID_tokens.png](../../assets/images/ama/howtos/configure-sso-8.png)

```

> 1. In the **Platform configurations** section, in **Web**, add additional URIs to the existing one:
> 
>     ![Add_redirect_URIs_additional.png](../../assets/images/ama/howtos/configure-sso-9.png)  
>     `https://departmentX.yourdomain.com`  
>     `https://app.yourdomain.com/auth/signin-oidc`

Reviewer: ```suggestion
1. In the **Platform configurations** section, in **Web**, add additional URIs to the existing one:

    `https://departmentX.yourdomain.com`
     
    `https://app.yourdomain.com/auth/signin-oidc`

    ![Add_redirect_URIs_additional.png](../../assets/images/ama/howtos/configure-sso-9.png)  
```
